phoenix/chain.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "terra2",
```

Comment:
This should match the folder name (I think `terra2` makes most sense, so changing the directory name)

Reply:
done in b6e7a91